Dockerfile

```diff
@@ -19,11 +19,11 @@ RUN set -ex \
     gunicorn \
     /buku[server] \
   && apk del .build-deps \
+  && echo "import sys;  bind = '0.0.0.0:${BUKUSERVER_PORT}'" > 'gunicorn.conf.py' \
```

Comment:
https://docs.gunicorn.org/en/stable/settings.html#config

(JSON format of `ENTRYPOINT` does not support variable interpolation)